src/query/storage/promremote/storage.go

```diff
@@ -269,6 +270,13 @@ func (p *promStorage) writeSingle(
 	}
 	req.Header.Set("content-encoding", "snappy")
 	req.Header.Set(xhttp.HeaderContentType, xhttp.ContentTypeProtobuf)
+	if endpoint.apiToken != "" {
+		req.Header.Set("Authorization", fmt.Sprintf("Basic %s",
+			base64.StdEncoding.EncodeToString([]byte(
+				fmt.Sprintf("%s:%s", tenant.name, endpoint.apiToken),
+			)),
```

Comment:
Is the header format documented somewhere on Grafana website? Add a piece of code comment with the url?

Reply:
No, I reverse engineered by the curl command you used: https://stackoverflow.com/questions/20737031/curlss-option-u